_data/simple-icons.json

```diff
@@ -14835,6 +14835,17 @@
             "hex": "000000",
             "source": "https://society6.com"
         },
+        {
+            "title": "Socket.dev",
```

Comment:
They call themselves `Socket`. Let's use `Socket` as title instead.

Reply:
That's probably better. I made this change.